### ISSUE TITLE
Drop support for Perl earlier than 5.26.x

### ISSUE THREAD

**Issue by ikedas** [sympa-community/sympa#1801]
Expected Behavior
-----------------

Sympa will support Perl 5.26.0 or later.

Current Behavior
----------------

Sympa has to support Perl 5.16.0 or later.

Additional information
----------------------
EPEL 7 repository that requires Perl 5.16.x will be [at end of life in 2024-06-30](https://pagure.io/epel/issue/238). _AFAIK_ the other distributions support later versions of Perl than 5.24.x.

My suggestion is that the next release of Sympa (maybe 6.2.74) will be the last version that supports Perl earlier than 5.26.

See also: #620, #1030.

**Comment by ldidry** [comment]
I agree to drop support of Perl < 5.24.x, but not for Sympa 6.2.74 as it should be released before the EOL of EPEL 7.
I suggest to drop support in the version after 6.2.74.

**Comment by ikedas** [comment]
I agree that 6.2.74 may support Perl <= 5.24.x.

**Comment by ikedas** [comment]
FYI.
[libintl-perl](/gflohr/libintl-perl/commit/91e1a391f7292d822af51b9e21bdce9e3bcefca9) (Locale::Messages) 1.34 drops support for Perl prior to 5.24.

**Comment by ikedas** [comment]
I'll merge the PR above in some days.

**Comment by alialobidm** [comment]
> أوافق على إسقاط دعم Perl < 5.24.x، ولكن ليس لـ Sympa 6.2.74 لأنه يجب إصداره قبل نهاية دعم EPEL 7. أقترح إسقاط الدعم في الإصدار الذي يلي 6.2.74
